login-form/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>login page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container flexcol">
+      <div class="loginform flexcol">
+        <div class="heading">Amplify</div>
+        <div class="fields desc">
+          <div class="uname">
+            <div><label>Username or Email</label></div>
+            <div><input type="text" class="username" /></div>
+            <div class="msgArea email"></div>
+          </div>
+          <div class="passwd">
+            <div><label>Password</label></div>
```

Comment:
why 2 `div` tags?

Reply:
passwd child to login form div and the label and its text box are children to passwd div, I'll see to it if I could reduce anything.